### ISSUE TITLE
Intero Python Module Not Found on Win32

### ISSUE THREAD

**Issue by lordmilko** [parsonsmatt/intero-neovim#83]
When attempting to use intero-neovim in Windows, an error is thrown when opening a Haskell file, stating

```
Error detected while processing function provider#python#Call:
line 18:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named intero
```

The issue is that python is unable to find the intero module defined in intero.py under the plugin's install directory. Running `:py print sys.path` confirms that the plugin directory was successfully added to the sys path [as expected](https://github.com/parsonsmatt/intero-neovim/blob/eed8a5ff3583e0fcf1d6f22cdb88e4731bdf01e1/autoload/intero/process.vim#L62).

```
['C:\\Windows\\SYSTEM32\\python27.zip', 'C:\\Python27\\DLLs', 'C:\\Python27\\lib',
'C:\\Python27\\lib\\plat-win', 'C:\\Python27\\lib\\lib-tk', 'C:\\Python27\\lib\\site-packages',
'_vim_path_', 'C:\\Users\administrator\\AppData\\Local\\nvim\\plugged\\intero-neovim']
```

However when executing `:py import intero` you can see with Process Monitor that Python doesn't actually search for the intero module on every folder on the sys.path. Rather, only the following directories are considered
* C:\Python27\DLLs
* C:\Python27\Lib
* C:\Python27\Lib\lib-tk
* C:\Python27
* C:\Python27\Lib\site-packages

Opening python.exe manually, adding the intero-neovim folder to the sys.path and importing the module successfully works (in that it gives an error it couldn't find the *vim* module)

Attempting to trick python into finding the intero module by symlinking from one of the directories it tries in gets past this step, however then we fail with other strange errors (which may or may not be due to our attempting to hack the system)

If a virtualenv is used instead of the global python install, python will search in both the virtualenv folder and the global folder, but still not the intero-neovim folder.

Minimal init.vim:

```vim
let g:python_host_prog='C:/Python27/python.exe'

call plug#begin('~/AppData/Local/nvim/plugged')

Plug 'parsonsmatt/intero-neovim'
Plug 'neomake/neomake'

call plug#end()
```

**Comment by lwm** [comment]
We've since added a `setup.py`, which may (totally guessing!) get the module on the `$PATH`. @lordmilko, could you try with the latest update? Also, do you see a possible fix for this (just skirting over this one, sorry if I missed something)?

**Comment by jez** [comment]
This should be fixed now that we no longer have Python (#91).
